libstatistics_collector/src/libstatistics_collector/collector/collector.cpp

```diff
@@ -12,16 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "collector.hpp"
 
 #include <mutex>
 #include <sstream>
 #include <string>
 
-#include "moving_average_statistics/moving_average.hpp"
-#include "moving_average_statistics/types.hpp"
+#include "libstatistics_collector/collector/collector.hpp"
+#include "libstatistics_collector/moving_average_statistics/moving_average.hpp"
+#include "libstatistics_collector/moving_average_statistics/types.hpp"
 
-namespace system_metrics_collector
+namespace collector
```

Comment:
dir and ns still wrong

Reply:
Since I'm on-call I most likely need to hand off this task. I put the work left to be completed in this [comment](https://github.com/ros-tooling/system_metrics_collector/pull/117#issuecomment-604142645) before your last review.

Reply:
So is this file supposed to be `libstatistics_collector/src/libstatistics_collector/libstatistics_collector/libstatistics_collector.cpp` now?

Reply:
That doesn't make a lot of sense... it seems like the last directory is just internal organization, it does not have to reflect the library name.